contents/handbook/people/team-structure/core-experience.md

```diff
@@ -3,4 +3,35 @@ title: Team Core Experience
 sidebar: Handbook
 showTitle: true
 hideAnchor: true
----
\ No newline at end of file
+---
+
+## People
+
+- [Eric Duong (Team Lead, Full Stack Engineer)](/handbook/company/team/#eric-duong-software-engineer)
+- [Paolo D'Amico (Product Manager)](/handbook/company/team/)
+- [Buddy Williams, Full Stack Engineer](/handbook/company/team/#buddy-williams-software-engineer)
+- Sam Winslow, Full Stack Engineer
+- [Li Yi Yu, Full Stack Engineer]((/handbook/company/team/#li-yi-yu-software-engineer))
+
+## Mission
+
+The most easy to use and useful analytics tool/experience to discover insights about your product and users
+
+## Responsibilities
+
+- Extending feature set as suggested by requests, by our own ideas, and by need for parity with other platforms
+- Maintaining data quality and clarity
+- Ensuring performant and clear user experience across all analytic functionality
+
+
+## Customer
+
+- Any PostHog user, whether they're an engineer or a product manager, should be able to effectively analyze their product.
+
+## Output metrics
+
+- Retention per feature
```

Comment:
Been giving this a lot of thought and have a few new ideas here, but might be better for a separate PR?

Regardless I think it'll be worth being super explicit about the metric, how is it measured, timeframe, ...